k-distribution/k-tutorial/1_basic/15_configurations/README.md

```diff
@@ -26,8 +26,12 @@ initialized with the value of the `$PGM` **configuration variable**.
 Configuration variables function as inputs to `krun`. These terms are supplied
 to `krun` in the form of ASTs parsed using a particular module. By default, the
 `$PGM` configuration variable uses the main syntax module of the definition.
+
 The cast on the configuration variable also specifies the sort that is used as
-the entry point to the parser, in this case the `K` sort.
+the entry point to the parser, in this case the `K` sort. In many situations it
+is useful to cast to other sorts there as well. The sort used for the `$PGM`
+variable is is referred to as the start symbol. During parsing, The default
+start symbol `K` subsumes all user-defined sorts except for syntactic lists.
```

Comment:
You have some typos:
`is is`, `parsing, The default`

Here is an alternative warding. Feel free to mix and match.

Normally, you would choose a sort from the language grammar for better control over the accepted language.
But in this case, `K` is used to specify all user-defined sorts except for syntactic lists.
Syntactic lists are excluded because they will always produce an ambiguity error when parsing a single element
and it would defeat the original purpose of being convenient.